src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -9930,6 +9930,56 @@ async Task<int> f()
                 Diagnostic(RudeEditKind.UpdatingStateMachineMethodAroundActiveStatement, "f"));
         }
 
+        [Fact]
+        [WorkItem(61415, "https://github.com/dotnet/roslyn/issues/61415")]
+        public void LocalFunction_EditAroundTry_WithActiveStatement()
+        {
+            var src1 =
+                """
```

Comment:
>""" [](http://example.com/codeflow?start=16&length=3)

nit: do raw strings actually make this better with all the indentation? Maybe I just got used to the previous style.

Reply:
In this case, with no quotes in the string, the only thing the raw string is providing is that the test input doesn't have to be based on column 0, so flows better in the IDE. The extra indentation is not present in the actual test input of course.

Reply:
Of course, though "flows better" is subjective here. I feel like I need to subtract the whitespace in my head when reading the embedded source code.

Reply:
Yeah, thats a good point. I'm very used to the vertical line in VS that takes away that pain.

Reply:
Yup, it's better in VS but sadly github/CodeFlow do not have it :(